catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -488,17 +491,19 @@ private void cloneSpriteVariables(Scene currentScene, Sprite cloneSprite) {
 		}
 	}
 
+	private void cloneLook(Sprite cloneSprite) {
+		cloneSprite.look = this.look.copyLookForSprite(cloneSprite);
+		if (cloneSprite.getLookDataList().size() > 0) {
+			cloneSprite.look.setLookData(cloneSprite.getLookDataList().get(0));
```

Comment:
Yes, definitely!
It's already done in https://github.com/Catrobat/Catroid/pull/1980 :wink: